donut/modules/newsgroups/templates/view_post.html

```diff
@@ -13,7 +13,18 @@ <h3>
 		<h3>Date: {{ post.time_sent }}</h3>
 		<h3>Subject: {{ post.subject }}</h3>
 		<h3>Message:</h3>
-		<h4>{{ post.message }}</h4>
+		<div id="msg"> </div>
 </div>
 </div>
 {% endblock %}
+{% block scripts %}
+	{{ super() }}
+	<script src="https://cdn.rawgit.com/showdownjs/showdown/1.8.7/dist/showdown.min.js"></script>
+	<script>
+		var target = $("#msg");
+		var post = {{ post.message | tojson }};
+      		converter = new showdown.Converter({strikethrough: true, tables: true});
```

Comment:
Usually better practice declare the converter, ie`var converter`
Also, indent's off